src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7461,7 +7523,7 @@ namespace UnitarySystems {
 
         if ((CoilType_Num == DataHVACGlobals::Coil_HeatingGasOrOtherFuel) || (CoilType_Num == DataHVACGlobals::Coil_HeatingElectric)) {
             HeatingCoils::SimulateHeatingCoilComponents(
-                state, CompName, FirstHVACIteration, _, this->m_SuppHeatCoilIndex, _, _, this->m_FanOpMode, this->m_SuppHeatPartLoadFrac);
+                state, CompName, FirstHVACIteration, _, this->m_SuppHeatCoilIndex, _, true, this->m_FanOpMode, this->m_SuppHeatPartLoadFrac);
```

Comment:
In addition to being used in sizing this flag also selects the report variable in ReportHeatingCoil. Probably should eliminate the Supp variable here.

        if (coilIsSuppHeater) {
            state.dataHVACGlobal->SuppHeatingCoilPower = heatingCoil.ElecUseLoad;
        } else {
            state.dataHVACGlobal->ElecHeatingCoilPower = heatingCoil.ElecUseLoad;
        }

Reply:
Well, can't get rid of that variable. These are used in Parent objects to sum the power for each component. So if there is a main heating coil and supplemental heating coil, these need to be kept separate.